src/wagtail_bynder/utils.py

```diff
@@ -24,11 +24,20 @@ def download_file(
     url: str, max_filesize: int, max_filesize_setting_name: str
 ) -> InMemoryUploadedFile:
     name = os.path.basename(url)
+    response = requests.get(url, timeout=20, stream=True)
+
+    # Make sure we don't store error responses instead of the file requested
+    if response.status_code != 200:
+        raise BynderAssetDownloadError(
+            f"Server error downloading '{name}' from Bynder. "
+        )
 
     # Stream file to memory
     file = BytesIO()
-    for line in requests.get(url, timeout=20, stream=True):
-        file.write(line)
+    for chunk in response.iter_content(chunk_size=8192):
```

Comment:
suggestion: this change could use some notes. Why not use the streamed get? why 8192 for the chunk size?